src/testing/jest/test/jest-config.spec.ts

```diff
@@ -19,6 +19,17 @@ describe('jest-config', () => {
     expect(jestArgv.maxWorkers).toBe(2);
   });
 
+  it('should pass --outputFile=path/to/file', () => {
+    const args = ['test', '--ci', '--outputFile=path/to/my-file'];
+    const config = mockValidatedConfig();
+    config.flags = parseFlags(args, config.sys);
+    expect(config.flags.args).toEqual(['--ci', '--outputFile=path/to/my-file']);
+    expect(config.flags.unknownArgs).toEqual([]);
+    config.testing = {};
```

Comment:
Minor, I think we can hoist this to be a part of the rest of the setup for this function:

```diff
  it('should pass --outputFile=path/to/file', () => {
    const args = ['test', '--ci', '--outputFile=path/to/my-file'];
    const config = mockValidatedConfig();
    config.flags = parseFlags(args, config.sys);
+    config.testing = {};
    expect(config.flags.args).toEqual(['--ci', '--outputFile=path/to/my-file']);
    expect(config.flags.unknownArgs).toEqual([]);
-    config.testing = {};
    const jestArgv = buildJestArgv(config);
    expect(jestArgv.outputFile).toBe('path/to/my-file');
  });
```